backend/LexBoxApi/Models/Project/CreateProjectInput.cs

```diff
@@ -12,5 +12,6 @@ public record CreateProjectInput(
     ProjectType Type,
     RetentionPolicy RetentionPolicy,
     bool IsConfidential,
+    Guid? OwningOrgId,
```

Comment:
Although our DB design allows projects to be owned by more than one org, I figured we don't want to assign multiple org owners *at project creation time*, so I left this as a single GUID rather than a list. Even if in the future we start to allow projects to be owned by multiple orgs in the UI, I doubt we'll ever want to create a project with two or more org owners at creation time, so even then we won't want this to be a list.